content/common/test/Makefile

```diff
@@ -0,0 +1,3 @@
+BINARY = open
```

Comment:
When building this, the output is:
```
gcc -g -Wall -Wextra -Werror -I../../common/makefile////../utils -I../../common/makefile////../utils/log  -c -o open.o open.c
gcc -g -Wall -Wextra -Werror -I../../common/makefile////../utils -I../../common/makefile////../utils/log  -c -o ../../common/makefile////../utils/log/log.o ../../common/makefile////../utils/log/log.c
gcc open.o ../../common/makefile////../utils/log/log.o  -o open
```

@teodutu , see if you can remove the extra slash (`/`) characters

Reply:
Done. `$(dir $(lastword $(MAKEFILE_LIST)))` retains the trailing `/`. The other variables that expanded this also added a `/`.